### ISSUE TITLE
VSCodium keeps menu bar on light mode

### ISSUE THREAD

**Issue by Mohammed-Balfakeih** [VSCodium/vscodium#1164]
I have installed VSCodium using https://www.linuxcapable.com/how-to-install-vscodium-on-fedora-36-linux/.

My Fedora 36 is in dark mode, yet when I boot up VSCodium the top bar is light, when I turn on "Auto Detect Color Scheme", it switches to the default light theme despite my system being in dark mode. The top bar remains the same. I would like to change the top bar to dark to suit my system more. I have not encountered a problem like this on any other software.

![image](https://user-images.githubusercontent.com/39861755/180003834-38005060-f172-44cb-829d-3243b01e110f.png)

**Comment by GitMensch** [comment]
That's very likely an upstream issue and then may be reported / checked at https://github.com/microsoft/vscode/issues/ (I guess you've searched there already, so creating a new issue there may be most useful?)

**Comment by Porkepix** [comment]
Experiencing the same issue here, and this looks unlikely this is a complete upstream issue: between two computers running ArchLinux, the old one runs vscode, the new one vscodium, in vscode the menubar is dark while on the new one vscodium have the menubar light (Gnome is in dark mode so the window decoration is black).

This is installed with https://aur.archlinux.org/packages/vscodium and https://archlinux.org/packages/community/x86_64/code/ respectively.

**Comment by uniboi** [comment]
Same issue here on Fedora 37. I checked vscodium and vscode and both have the top bar in light mode.

**Comment by Porkepix** [comment]
Okay, so I found out why on one computer this was dark while on the other it was light: the one where it was dark is an old installation and it turns out I had selected adwaita (dark) in Gnome Tweaks (https://gitlab.gnome.org/GNOME/gnome-tweaks) which affects vscode(ium). Setting this to default turns back every GTK element to light.
It looks like to have this natively without any hack, this would be needed: https://github.com/electron/electron/issues/33635 (seen from https://github.com/microsoft/vscode/issues/146804)
And in the ends it's indeed a vscode and even rather an electron issue.
